src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala

```diff
@@ -89,4 +87,17 @@ object RoutesRepoHelper extends DaoHelper {
         |      }]
         |  }]
         |}""".stripMargin
+
+  private def insertTestPath(ownedByTeam: String, createdBy: String, createdAt: LocalDateTime): Long = {
```

Comment:
for 3 or less parameters I think the readability is better if we keep them on the same line